### PR TITLE
Adds Custom Mentor role, can Add Wiki and Stuff now

### PR DESCRIPTION
**Pull request by JamieD1**
Adds Custom Mentor role, can Add Wiki and Stuff now

### CODE REVIEW

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Update SQL schema

**Comment by alexkar598** [comment]
For the record, storing the rank as a string across all rows is a stupid idea

**Comment by adamsong** [comment]
How would you recommend storing it?

**Comment by alexkar598** [comment]
As an id which points to a rank

**Comment by adamsong** [comment]
Admin ranks use a string

**Comment by alexkar598** [comment]
Whoever came up with that idea doesn't know how a relational database works

**Comment by Bibby0110** [comment]
> Whoever came up with that idea doesn't know how a relational database works

our codebase in a nutshell

File: SQL/database_changelog.txt [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 
-INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 4);
+INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 5);
 or
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 4);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 5);
 
 In any query remember to add a prefix to the table names if you use one.
 
+version 5.5 14 August 2021, by JamieD1
```
Comment (Bibby0110) [inline SQL/database_changelog.txt:13]: add 
 \---------------------------------------------------- 
above

**Comment by N3D6** [comment]
Maybe needs a bug report but right now rementoring after dementoring makes your role show up as [] when you talk in OOC

**Pull request merged**